net.sapples.LiveCaptions.json

```
@@ -0,0 +1,108 @@
+{
+    "app-id" : "net.sapples.LiveCaptions",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "livecaptions",
+    "finish-args" : [
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--socket=fallback-x11",
+        "--env=LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/app/lib"
```

```suggestion
        "--socket=fallback-x11"
```

you don't need to set this.